### DISCUSSION TITLE
Kick off the conversion tasks in order of the watch folders

### DISCUSSION THREAD

**Discussion by mkjustuk** [jlesage/docker-handbrake#323] [Ideas]
I have three watch folders that are all the same dir, but use different presets.  This gives me 3 outputs in differing quality for the same source file.  Each conversion kicks off in turn, but it is not always in the logical order of watch folder 1,2 then 3.

Could there be an env variable added that sets the order (e.g. WATCH_FOLDER_ORDER) if a value of 1,3,4,2 is set?

**Comment by jlesage** [comment]
Looks like you could just re-map the watch folders to get the wanted ordering ?

**Reply by mkjustuk** [reply]
Thanks for the reply.  This is how I have it, but the order varies.

I have the following container volumes mapped to the same host DIR (where I drop the source files) - watch, watch2 and watch3
I then have different presets as ENV variables for AUTOMATED_CONVERSION_PRESET, AUTOMATED_CONVERSION_PRESET_2 and AUTOMATED_CONVERSION_PRESET_3

I'd expect the order to always be watch, watch2 and then watch3 but often it's actually watch3, watch and then watch2.  Sometimes I see watch2, watch and then watch3.
